homework-2/src/templates/what.html

```diff
@@ -0,0 +1,13 @@
+<div class="what">
```

Comment:
what if the text inside will be changed? You call section by first word of title